silverback/settings.py

```diff
@@ -17,6 +18,9 @@ class Settings(BaseSettings, ManagerAccessMixin):
     testing or deployment purposes. Defaults to a working in-memory broker.
     """
 
+    # A unique identifier for this silverback instance
+    INSTANCE: str = "default"
```

Comment:
What's the idea behind this?

Reply:
The ability for multiple Silverback instances to use a single persistence backend.  For ApePay, I'm setting it to apepay-sepolia, and we may have others for other networks.  Otherwise, they'd step on each-others toes and confusingly share state.